modules/gathering-crio-logs.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-crio-issues.adoc
+
+[id="gathering-crio-logs_{context}"]
+= Gathering CRI-O journald unit logs
+
+If you experience CRI-O issues, you can obtain CRI-O journald unit logs from a node.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the cluster-admin role.
+* Your API service is still functional.
+* You have installed the OpenShift CLI (`oc`).
+* You have the fully-qualified domain names of the control plane, or master machines.
```

Comment:
s/fully-qualified/ fully qualified/ 
Do not use a hyphen after an adverb or adjective that ends in _-ly_ because such terms are
never ambiguous.

Reply:
Resolved throughout the PR.